prov/rxm/src/rxm.h

```diff
@@ -396,7 +398,7 @@ struct rxm_ep {
 
 	struct rxm_buf_pool	buf_pools[RXM_BUF_POOL_MAX];
 
-	struct dlist_entry	post_rx_list;
+	struct dlist_entry	srx_posted_rx_list;
```

Comment:
nit: this could just be named posted_srx_list.  Can the name change go into a separate commit?